### PR TITLE
fix docgen crappiness

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers

### CODE REVIEW

File: shared/desktop/yarn-helper/index.js [hunk 1]
```diff
@@ -26,11 +28,27 @@ const commands = {
   postinstall: {
     code: () => {
       // Nothing right now
```
Comment (MarcoPolo) [inline shared/desktop/yarn-helper/index.js]: This comment doesn't make sense anymore

File: shared/desktop/yarn-helper/index.js [hunk 2]
```diff
@@ -26,11 +28,27 @@ const commands = {
...
+function makeShims() {
+  const root = path.resolve(__dirname, '..', '..', 'node_modules', 'babel-plugin-react-docgen')
+
+  try {
+    fs.mkdirSync(root)
+  } catch (_) {}
+
+  try {
+    fs.writeFileSync(path.join(root, 'package.json'), `{"main": "index.js"}`)
+    fs.writeFileSync(path.join(root, 'index.js'), `module.exports = function(){return {};};`)
```
Comment (MarcoPolo) [inline shared/desktop/yarn-helper/index.js]: Nothing expects this to exist?

Comment (chrisnojima) [reply shared/desktop/yarn-helper/index.js]: ?

Comment (MarcoPolo) [reply shared/desktop/yarn-helper/index.js]: I mean that you're effectively removing this file, right? so why doesn't that break anything else?

Comment (chrisnojima) [reply shared/desktop/yarn-helper/index.js]: only story book needs this so this plugin does nothign now. we don't use it. also it uses lib/index.js and not index.js

**Pull request merged**